KarhooUISDK/Screens/PrebookConfirmationScreen/KarhooBookingConfirmationViewModel.swift

```diff
@@ -174,6 +176,14 @@ class KarhooBookingConfirmationViewModel: BookingConfirmationViewModel {
         return dateFormatter.display(clockTime: journeyDetails.scheduledDate)
     }
 
+    var accessibilityDate: String {
+        dateFormatter.display(fullDate: journeyDetails.scheduledDate)
```

Comment:
According to ticket A/C this text should be more descriptive, like: "Pick-up time 05:11 PM Tuesday 26 july 2022"